### PR TITLE
Change A5 S5 to loosen org restrictions

### PR DESCRIPTION
**Pull request by mathmitch7**
The idea is that the old org classification rules are too strict to fund all things that have 'significant or consistent' funding needs. As such, I'm changing the 10 person 1 year rule to be mere suggestions.

### CODE REVIEW

File: student_government_bylaws.md [hunk 1]
```diff
@@ -564,4 +564,4 @@ Additionally, each Transparency Document must contain the name of a current memb
 
 #### Section 3. Call to Action.
 
-An annual Call to Action will be sent to the Student Body in the first two weeks of classes each school year. This Call to Action will inform the Student Body of the core-discussion mailing list, of the locations of the Transparency Documents, of the elected members of the Student Government, and of the planned open meeting times of the Student Government and its Committees.
```
Comment (MattWis) [inline student_government_bylaws.md]: This line doesn't appear to have actually been changed. Am I missing something?

Comment (mathmitch7) [inline student_government_bylaws.md]: It seems like nothing was changed. Hm.

Comment (iangmhill) [inline student_government_bylaws.md]: Should it have been changed?

Comment (mathmitch7) [inline student_government_bylaws.md]: Nope

**Comment by iangmhill** [comment]
Amendment approved by the Student Government on November 6, 2015.

**Pull request merged**